static/src/javascripts/projects/commercial/modules/dfp/prepare-prebid.spec.js

```diff
@@ -110,6 +134,28 @@ describe('init', () => {
         await setupPrebid();
         expect(prebid.initialise).toBeCalled();
     });
+    it('should initialise Prebid if TCFv2 consent with correct Sourcepoint Id is true in variant', async () => {
+        dfpEnv.hbImpl = { prebid: true, a9: false };
+        commercialFeatures.dfpAdvertising = true;
+        commercialFeatures.adFree = false;
+        isInVariantSynchronous.mockImplementation(
+            (testId, variantId) => variantId === 'variant'
+        );
+        shouldUseSourcepointCmp.mockImplementation(() => true);
+        onConsentChange.mockImplementation(tcfv2WithConsentMock);
+        await setupPrebid();
+        expect(prebid.initialise).toBeCalled();
+    });
+
+    it('should not initialise Prebid if TCFv2 with correct Sourcepoint Id is false', async () => {
+        dfpEnv.hbImpl = { prebid: true, a9: false };
+        commercialFeatures.dfpAdvertising = true;
+        commercialFeatures.adFree = false;
+        shouldUseSourcepointCmp.mockImplementation(() => true);
+        onConsentChange.mockImplementation(tcfv2WithoutConsentMock);
+        await setupPrebid();
+        expect(prebid.initialise).not.toBeCalled();
+    });
```

Comment:
You might also want to test Prebid being initialised if Sourcepoint ID is `false` but user is **not** in variant.

Reply:
That's what this test was doing, indirectly by not mocking the `isInVariantSynchronous` to a `variant` value (falling back to undefined). 
I have now made it more explicit by adding a mock value `notintest` instead